components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -177,8 +181,15 @@ public void publishToGateway(API api, APITemplateBuilder builder, String tenantD
 					deployCustomSequences(api, tenantDomain, environment);
 				}
 			}
-		}
-	}
+            } catch (AxisFault axisFault) {
+                log.error("Error occurred when publish to gateway " + environmentName, axisFault);
```

Comment:
Put comments and explain why we are logging and continuing instead of throwing the exception.